setup.py

```diff
@@ -74,7 +74,7 @@
         "Topic :: System :: Monitoring",
         ],
     install_requires=[
-        'requests<=2.10.0',
+        'requests>=0.12.1,<=2.10.0',
```

Comment:
I don't think we should only do the `<=` for python versions where this is needed. Doing so exposes us to any breaking changes that the requests maintainers make. 

E.g. A user is on Python 3.1 and we are not specifying `<=` for the requests package. Then, the requests maintainers add some functionality which breaks things for Python 3.1. We will have no way of knowing that this happens for our customers unless we re-run Travis tests. 

By restricting the range to what we know works right now we are playing it safe for future changes made to the requests library.